src/lib/DepositRecordSerializer.js

```diff
@@ -15,11 +15,7 @@ import _pickBy from 'lodash/pickBy';
 import _pick from 'lodash/pick';
 import _mapValues from 'lodash/mapValues';
 import { emptyCreator, emptyContributor } from './record';
-import {
-  Field,
-  ContributorsSerializer,
-  CreatorsSerializer,
-} from './serializers';
+import { Field, ContributorsField, CreatorsField } from './fields';
 
 export class DepositRecordSerializer {
   defaultRecord = {
```

Comment:
Do you mean as fallback defaults if one of the `defaultDeserializedValue`, `defaultSerializedValue` is not passed?

Reply:
I am a bit hesitant on that because I am thinking of users that want to extend the `DepositRecordSerializer` class. It's much simpler if they just extend and override the `defaultRecord` and thus change the defaultValues passed in the fields...